modules/andes-core/broker/src/main/java/org/wso2/andes/server/handler/ChannelFlowHandler.java

```diff
@@ -81,6 +81,6 @@ public void onException(Exception exception) {
 
             }
         });
-        logger.info(" Channel flow: " + body.getActive() + " set for channel: " + channel.getId());
+        logger.debug(" Channel flow: " + body.getActive() + " set for channel: " + channel.getId());
     }
```

Comment:
Shall we call logger.isDebugEnabled() before logger.debug() since the log message contains some string concatenation?